File: Services/CategoryServices/DrivingSchoolApp.Services.CategoryServices.BCategoryAPI/DataAccerrss/Entities/Instructor.cs
namespace DrivingSchoolApp.Services.CategoryServices.BCategoryAPI.DataAccerrss.Entities
{
    public class Instructor : Base
    {
		public string Firstname { get; set; }
		public string Lastname { get; set; }
		public int Age { get; set; }
		public string LicenceId { get; set; }
		public string Photo { get; set; }
		public string StartedDay { get; set; }
		public int TotalWorkExperience { get; set; }
		public int DriveExperience { get; set; }
		public string InstructorAcademy { get; set; }
		public bool isBooked { get; set; }
		public int CategoryBId { get; set; }
		public virtual CategoryB CategoryB { get; set; }
		public IEnumerable<WorkExperience> WorkExperiencesPerCompany { get; set; } = new List<WorkExperience>();
	}
}

File: Frontend/DrivingSchoolWeb/src/app/categories/category-b/models/instructor.ts
import {Base} from 'src/app/categories/shared/models/base';
import {WorkExperience} from 'src/app/categories/category-b/models/work-experience';

export interface Instructor extends Base{
  fullname: string,
  age: number,
  licenceId: string,
  photo: string | null,
  startedDay:string,
  totalWorkExperience:number,
  driveExperience:number,
  instructorAcademy:number,
  isBooked:boolean,
  categoryId:number,
  workExperiencePerCompany:WorkExperience[]
}
